velox/docs/README.md

```diff
@@ -10,9 +10,22 @@ To install Sphinx: `easy_install -U sphinx`
 
 `sphinx-quickstart` command was used to generate the initial Makefile and config.
 
+### Building PyVelox Components
+
+Doxygen is used to generate the C++ documentation in language bindings. 
+Refer to installation instructions [here](https://www.doxygen.nl/manual/install.html).
+If you're using a conda environment, it can be easily installed by `conda install` command.
+
+Pandoc is also used to generate `.rst` files from existing markdown files. Refer to installation
+instructions [here](https://pandoc.org/installing.html).
+
+[Breathe](https://github.com/breathe-doc/breathe) is also required in generating the binding documentation.
+
+Generate Python bindings Get started page.
```

Comment:
This sentence seems disconnected from the rest of the text. Would you double check?

Reply:
Ah yes, good catch, I have missed removing this since we removed the doxygen component in a review. I will update the PR.